docarray/typing/tensor/ndarray.py

```diff
@@ -202,11 +202,11 @@ def to_protobuf(self) -> 'NdArrayProto':
         return nd_proto
 
     @staticmethod
-    def get_comp_backend() -> Type['NumpyCompBackend']:
+    def get_comp_backend() -> 'NumpyCompBackend':
         """Return the computational backend of the tensor"""
         from docarray.computation.numpy_backend import NumpyCompBackend
 
-        return NumpyCompBackend
+        return NumpyCompBackend()
```

Comment:
Is it a bit of an anti pattern to crate all of these instances if we don't really need them? would it make sense to maintain a singleton instance?

Reply:
Oh that is a good idea actually. I support this. Lets go singleton with this.

Reply:
actually doing the singleton is not that trivial in python ... I think it adds complexity for few real advantages at the end. Only conceptual advantage.